### ISSUE TITLE
How to display a history chart for multiple symbols?

### ISSUE THREAD

**Issue by mrpaulharris** [dpguthrie/yahooquery#123]
What is the command to display the chart in https://yahooquery.dpguthrie.com/guide/ticker/historical/

![image](https://user-images.githubusercontent.com/71358025/197361020-2427ce07-c7b9-4ace-a55f-d95fa9c9c816.png)

I tried this code

```
import pandas as pd
import matplotlib.pyplot as plt
from yahooquery import Ticker

tickers = Ticker('aapl nflx', asynchronous=True)
df = tickers.history()
df["adjclose"].plot()
plt.xticks(rotation=90)
plt.show()
```
But it is just showing one series in the chart, like this:

![image](https://user-images.githubusercontent.com/71358025/197361201-ee22e3da-da06-4784-818f-a88550222707.png)

How can I create aapl and nflx as two series on the one chart?

**Comment by dpguthrie** [comment]
This should do it:

```python
import matplotlib.pyplot as plt
from yahooquery import Ticker

t = Ticker('aapl nflx', asynchronous=True)
df = t.history()
df.reset_index().set_index('date').groupby('symbol')['adjclose'].plot(legend=True, xlabel='Date', ylabel='adjclose')
plt.show()
```

**Comment by mrpaulharris** [comment]
Thanks, that worked great, very clever. 

I also came across this solution, using `pivot()` which may benefit some of your users.

```
import pandas as pd
import matplotlib.pyplot as plt
from yahooquery import Ticker

t = Ticker('^AXJO AKE.AX', asynchronous=True)
df = t.history()

df =  df[~df.index.duplicated(keep='first')]
df_pivot = df.reset_index().pivot(index='date', columns='symbol', values='adjclose')
df_pivot_perc = df_pivot.div(df_pivot.iloc[0,:]).mul(100)
df_pivot_perc.plot(legend=True, xlabel='Date', ylabel='adjclose')

plt.xticks(rotation=90)
plt.show()
```
Notes:
1. It uses `pivot()` to change the shape of the dataframe.
2. It resets values to be a percentage, so they are easier to compare.
3. I also found some stocks returned a duplicate date from yahooquery, so I removed them with `duplicated` function.

**PS: Is the duplicate date a possible bug?**

Ticker gives the last date for AKE.AX twice. With same prices, but with different volumes. Interestingly, you don't get this error with import yfinance as yf and then ake = yf.Ticker("AKE.AX") and hist = ake.history(period="ytd"). You'll get just 1 entry for 2022-10-21 with the same volume as given through Ticker in the last row.
